x/distribution/keeper/hooks.go

```diff
@@ -77,6 +78,14 @@ func (k Keeper) onDelegationCreated(ctx sdk.Context, delAddr sdk.AccAddress,
 func (k Keeper) onDelegationSharesModified(ctx sdk.Context, delAddr sdk.AccAddress,
 	valAddr sdk.ValAddress) {
 
+	if bytes.Equal(delAddr.Bytes(), valAddr.Bytes()) {
+		// On updates to a self bond/unbond, we must update the validator's dist
+		// info without withdrawing any rewards.
+		k.UpdateValidatorDistInfoFromPool(ctx, valAddr)
```

Comment:
this doesn't need to be exposed (should keep consistency)

Reply:
Sure! Updated. 

But consistency with what exactly? Everything in `x/distribution/keeper/validator.go` is exposed (which is where this lives). I wasn't able to find a consistent pattern for these methods. I think a large bulk of them should be unexposed.